payloads/ais_target.json

```
@@ -0,0 +1,129 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "http://example.com/example.json",
+    "type": "object",
+    "title": "The root schema",
+    "description": "The root schema comprises the entire JSON document.",
+    "default": {},
+    "examples": [
+        {
+            "mmsi": 124716291,
+            "imo_number": 214617214,
+            "draught": 1241.782,
+            "type_and_cargo": 23,
+            "ship_name": "Boaty McBoatface",
+            "call_sign": "OWIP2",
+            "destination": "Gothenburg",
+            "eta": "2021-10-07T20:20:39.345+00:00",
+            "length": 123,
+            "beam": 12,
+            "pose": {},
+            "twist": {}
+        }
+    ],
+    "required": [
+        "mmsi",
+        "imo_number",
+        "draught",
+        "type_and_cargo",
+        "ship_name",
+        "call_sign",
+        "destination",
+        "eta",
+        "length",
+        "beam",
+        "pose",
+        "twist"
+    ],
+    "properties": {
+        "mmsi": {
+            "type": "integer",
+            "title": "MMSI",
+            "description": "MMSI number of the AIS transceiver",
+            "examples": [
+                124716291
+            ]
+        },
+        "imo_number": {
+            "type": "integer",
+            "title": "IMO number",
+            "description": "IMO number of the vessel",
+            "examples": [
+                214617214
+            ]
+        },
+        "draught": {
+            "type": "number",
+            "title": "Draught",
+            "description": "Draught of this vessel",
+            "examples": [
+                12.6
+            ]
+        },
+        "type_and_cargo": {
```

Suggested change to better match the [AIS spec](https://www.itu.int/rec/R-REC-M.1371-5-201402-I/en) 
```suggestion
        "type_of_vessel": {
```